glmtools/tests/test_glm.py

```diff
@@ -16,4 +17,33 @@
 def test_glm_t_test():
     # Test glm and t_test against scipy
     # Your test code here
+
+    # Testing obvious things
```

Comment:
Nice - good example of testing things that you know should be true in simple cases.